src/mpol/coordinates.py

```diff
@@ -131,7 +131,7 @@ def __init__(self, cell_size: float, npix: int) -> None:
         self.packed_q_centers_2D = np_fft.fftshift(self.sky_q_centers_2D)
         self.packed_phi_centers_2D = np_fft.fftshift(self.sky_phi_centers_2D)
 
-        self.q_max = (
+        self.q_max = float(
```

Comment:
Just curious, what's the reason for the float call? Shouldn't this already be as a float type?

Reply:
Some of the numpy fft calls end up returning values that the linter can't infer its type from. The float call just makes it explicit for use with mypy later.